### ISSUE TITLE
Not enough merge conflicts

### ISSUE THREAD

**Issue by DeltaF1** [seanrt/GitWorkshopIEEECarleton#35]
This repository needs more merge conflicts.

**Comment by ecmjohnson** [comment]
I agree. We should outsource to students; I hear they are great at creating a lot of merge conflicts without us needing to pay them much.
